src/libs/Accessibility/index.js

```diff
@@ -0,0 +1,44 @@
+import {useEffect, useState, useCallback} from 'react';
+import {AccessibilityInfo} from 'react-native';
+import _ from 'underscore';
+import moveAccessibilityFocus from './moveAccessibilityFocus';
+
+const useScreenReaderStatus = () => {
+    const [isScreenReaderEnabled, setIsScreenReaderEnabled] = useState(false);
+    useEffect(() => {
+        const subscription = AccessibilityInfo.addEventListener('screenReaderChanged', setIsScreenReaderEnabled);
+
+        return subscription.remove;
+    }, []);
+
+    return isScreenReaderEnabled;
+};
+
+const getHitSlopForSize = ({x, y}) => {
+    const minimumSize = 44;
```

Comment:
Could you add a comment explaining _why_ 44 is used as the "magic number" here?

Reply:
Sure, done.